src/game/client/draw/drawable.cpp

```diff
@@ -3069,3 +3083,211 @@ void Drawable::Apply_Physics_Xform(Matrix3D *mtx)
         }
     }
 }
+
+void Drawable::Set_Fully_Obscured_By_Shroud(bool fully_obscured)
+{
+    for (DrawModule **i = Get_Draw_Modules(); *i != nullptr; i++) {
```

Comment:
needs to check m_fullyObscuredByShroud before

Reply:
fixed.